### PR TITLE
DPD-279 benchmarks

### PR DESCRIPTION
**Pull request by Getriax**
https://littledata.atlassian.net/browse/DPD-279

### CODE REVIEW

File: test/gaTracker/helpers.test.ts [hunk 1]
```diff
@@ -1,27 +0,0 @@
...
-const fakePageURL = '/test';
-
-describe('gaHelpers', () => {
-	before(() => {
-		const gtagStub = '';
-
-		global.gtag = sinon.stub(null, gtagStub);
-	});
-	context('checking the pageview', () => {
-		it('should send the email hashed to dL if customer object available');
```
Comment (igorkol) [inline test/gaTracker/helpers.test.ts:25]: please don't delete the file. it has a temporary placeholder role and is a reminder that we need to write a whole bunch of tests for the helpers.ts file

**Review by igorkol — CHANGES_REQUESTED** [review]
some minor changes needed. otherwise it's okay

File: test/gaTracker/helpers.test.ts (hunk 1 above)
Comment (Getriax) [reply test/gaTracker/helpers.test.ts]: Restored

File: src/gaTracker/helpers.ts [hunk 2]
```diff
@@ -186,26 +189,29 @@ export const trackEvents = (): void => {
...
-						...ga4EventParams(),
-						method: network,
-					});
+					sendGa4Event(
+						'share',
+						{
+							...ga4EventParams(),
+							method: network,
+						},
+						{ event_label: network },
```
[the remark this answers is not in the document]
Comment (igorkol) [reply src/gaTracker/helpers.ts:198]: okay then. i suppose we can easily add it in when/if we see the data missing in the benchmarks

**Review by igorkol — APPROVED** [review]

**Pull request merged**